dunwoody_disaster/CharacterFactory.py

```diff
@@ -162,81 +168,17 @@ class CharacterFactory:
             "level": 1,
             "loot": [],
             "food": [],
-        },
-        "warrior": {
-            "health": 100,
-            "magic": 0,
-            "stamina": 15,
-            "strength": 15,
-            "intelligence": 5,
-            "defense": 0,
-            "magicDefense": 0,
-            "level": 1,
-            "loot": [],
-            "food": [],
-        },
-        "mage": {
-            "health": 100,
-            "magic": 15,
-            "stamina": 0,
-            "strength": 5,
-            "intelligence": 15,
-            "defense": 0,
-            "magicDefense": 0,
-            "level": 1,
-            "loot": [],
-            "food": [],
-        },
-        "thief": {
-            "health": 100,
-            "magic": 0,
-            "stamina": 10,
-            "strength": 8,
-            "intelligence": 8,
-            "defense": 0,
-            "magicDefense": 0,
-            "level": 1,
-            "loot": [],
-            "food": [],
-        },
+        }
     }
 
     @staticmethod
-    def createCharacter(name: str, classType: str) -> Character:
+    def createCharacter() -> Character:
```

Comment:
Might aswell just remove this function.

Reply:
I think we need this until we figure out how we are giving the players their weapons. That's currently the only way we have weapons.

Reply:
Oh, right.